spanner/opentelemetry/tracing/Readme.md

```diff
@@ -0,0 +1,49 @@
+# Cloud Spanner OpenTelemetry Traces
+
+## Setup
+
+This sample requires [Go](https://go.dev/doc/install).
+
+1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/go/docs/setup).**
+
+2.  Enable APIs for your project.
+
+    a. [Click here](https://console.cloud.google.com/flows/enableapi?apiid=spanner.googleapis.com&showconfirmation=true)
+    to visit Cloud Platform Console and enable the Google Cloud Spanner API.
+
+    b. [Click here](https://console.cloud.google.com/flows/enableapi?apiid=cloudtrace.googleapis.com&showconfirmation=true)
+    to visit Cloud Platform Console and enable the Cloud Trace API.
+
+3.  Create a Cloud Spanner instance and database via the Cloud Plaform Console's
+    [Cloud Spanner section](http://console.cloud.google.com/spanner).
+
+4.  Enable application default credentials by running the command `gcloud auth application-default login`.
+
+## Run the Example
+
+1. Set up database configuration in the `spanner_opentelemetry_tracing.go` file:
+    ````
+    var projectId = "projectId"
```

Comment:
Nit: by convention in Go, the identifiers should be `projectID`, `instanceID`, `databaseID`